app/models/season.rb

```diff
@@ -38,8 +38,8 @@ def projects_proposable?
         season.project_proposals_close_at
     end
 
-    def all_years
-      pluck(:name)
+    def active_and_previous_years
+      where("acceptance_notification_at <= ?", Time.now.utc).order(:name).pluck(:name)
```

Comment:
Hey @agpiermarini Great!! I love your solution using the general `acceptance_notification` date!!

One more thing, and sorry I missed that in the previous review. The `order` will return the season names as `oldest season first`, right? And it really should return the latest season first, = `:desc`.
Would you please add that? Either leave it in this query, or put it in a named scope `ordered_by_name` or something.

Reply:
Absolutely. I'm inclined to leave it in the query because the reverse chronological ordering does not seem to be used in any other Season model queries, and it may be a bit more readable to have everything in one place. With that said, if your preference would be to scope the ordering operation, I'd be happy to do so.